src/Rest_API/Shipping/Item_Info.php

```diff
@@ -917,4 +921,21 @@ public function get_product_options() {
 
 		return array();
 	}
+
+	/**
+	 * Check mailbox weight limit.
+	 *
+	 * @param $backend_data  .
+	 * @param $order_weight  .
+	 *
+	 * @return void.
+	 * @throws \Exception if the order weight exceeds 2000 grams.
+	 */
+	protected function check_mailbox_weight_limit( $backend_data, $order_weight ) {
+		if ( 'yes' === $backend_data['mailboxpacket'] && $order_weight > 2000 ) {
+			throw new \Exception(
+				__( 'Max weight for Mailbox is 2 kg!', 'postnl-for-woocommerce' )
+			);
+		}
```

Comment:
```suggestion
		if ( 'yes' === $backend_data['mailboxpacket'] && 2000 < $order_weight ) {
			throw new \Exception(
				esc_html__( 'Max weight for Mailbox is 2 kg!', 'postnl-for-woocommerce' )
			);
		}
```
let's keep the Yoda style consistent for the condition and escaping the text there.